query/dev/system/src/main/java/org/codice/ddf/admin/query/dev/system/graph/DependencyEdge.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.query.dev.system.graph;
+
+public class DependencyEdge<V> {
+  private V value;
+
+  public DependencyEdge(V value) {
```

Comment:
Any reason we have this as public + a static factory method? Maybe make this private?

Reply:
Is there really value in hiding the constructor?

Reply:
If you're providing a factory then you should hide the ctor to direct all access through the factory.